src/common/snippets/include/snippets/lowered/loop_manager.hpp

```diff
@@ -176,8 +208,13 @@ class LinearIR::LoopManager {
                      size_t increment,
                      size_t dim_idx,
                      const std::vector<T>& entries,
-                     const std::vector<T>& exits) {
-        const auto loop_info = std::make_shared<LoopManager::LoopInfo>(work_amount, increment, entries, exits);
+                     const std::vector<T>& exits,
+                     bool set_default_handlers = true) {
```

Comment:
May I ask you to update brief descriptions for methods in this class please?

Reply:
The brief descriptions of `mark_loop` methods were updated, thanks for the reminder